src/utils/getDataFirstTime.ts

```diff
@@ -0,0 +1,16 @@
+interface Params {
+  data?: any[];
+  isLoading?: boolean;
+  getDataFunction: (data?: any) => any;
+}
+
+export async function getDataFirstTime({
```

Comment:
It doesn't make sense. Redundant nesting. And utils are for general-purpose helper functions only